### ISSUE TITLE
How to add props such as autoComplete to the form?

### ISSUE THREAD

**Issue by petercinibulk** [vazco/uniforms#1177]
I could not find any documention or issues relating to this so hopefully someone can answer this. BTW this library is amazing :)

I want to be able to do something like this with uniforms:
```
export default function App() {
  return (
    <div className="App">
      <form autoComplete="off">         <input type="text"/>
      </form>
    </div>
  );
}
```
Such as:
```
import React from 'react';
import { AutoForm } from 'uniforms-semantic';

import { bridge as schema } from './GuestSchema';

export function GuestFormBasic() {
  return <AutoForm schema={schema} onSubmit={console.log} autoComplete="off" />;
}
```
Is there a way to access the internal form tag?

Thanks!

**Comment by radekmie** [comment]
Hi @petercinibulk! These should be passed properly, even if the types say otherwise.
* If there's a problem with that, do let us know.
* Types will be fixed in #1160.

**Comment by petercinibulk** [comment]
Awesome thanks! I am using TS and ESLint so I had to configure it to allow this
